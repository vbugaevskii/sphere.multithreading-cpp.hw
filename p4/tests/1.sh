  echo     test1  test2
echo

echo test3
