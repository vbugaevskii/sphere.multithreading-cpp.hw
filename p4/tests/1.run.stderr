Process 13921 exited: 0
Process 13922 exited: 0
Process 13923 exited: 0
